src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -244,6 +270,8 @@ public boolean equals(final @Nullable Object obj) {
 		types.add(new SimpleEntityDataInfo("projectile", Projectile.class, true));
 		types.add(new SimpleEntityDataInfo("living entity", LivingEntity.class, true));
 		types.add(new SimpleEntityDataInfo("entity", Entity.class, true));
+		types.add(new SimpleEntityDataInfo("water mob" , WaterMob.class, true));
```

Comment:
Do these really exist before 1.13?

Reply:
Apparently so; [here's](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/commits/b39503de71d00627a9486c98f40ae235a6299151#src/main/java/org/bukkit/entity/WaterMob.java) an example of a commit referencing it as early as 2011 (by Grum, actually)

Reply:
I guess it would be very useful in the 1.13 update, since there's so many water mobs now.... if you were wanting to use a spawn event for all water mobs and cancel it, or whatever a Skripter is wanting to do with them.